src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -82,6 +87,7 @@ export default class DataTable extends Vue {
     ? this.$props.config.showSelectAllDropdown
     : defaultConfig.showSelectAllDropdown;
   printMode = this.$props.config?.print?.mode || defaultConfig.print?.mode;
+  emptyMessage = this.config.noFiltersMessage || defaultConfig.noFiltersMessage || DATA_TABLE_NO_FILTERS_MESSAGE;
```

Comment:
this.config.noFiltersMessage and defaultConfig.noFiltersMessage are optional
So that's why there is DATA_TABLE_NO_FILTERS_MESSAGE to avoid TS messages about undefined values